src/stream.ts

```diff
@@ -213,6 +213,7 @@ export class WebRTCStream implements Stream {
 
   constructor (opts: StreamInitOpts) {
     this.channel = opts.channel
+    this.channel.binaryType = 'arraybuffer'
```

Comment:
It should be since the default is supposedly a blob: https://developer.mozilla.org/en-US/docs/Web/API/RTCDataChannel/binaryType. However I see the binaryType as arraybuffer when I create the channel. So I'm not sure why. I think we should keep this anyways.